lib/web/apiserver.go

```diff
@@ -1687,6 +1689,7 @@ func (h *Handler) getWebConfig(w http.ResponseWriter, r *http.Request, p httprou
 	isTeam := clusterFeatures.GetProductType() == proto.ProductType_PRODUCT_TYPE_TEAM
 
 	webCfg := webclient.WebConfig{
+		IsCommunity:                    teleportEdition == "community",
```

Comment:
Note that `lib/modules` already exposes a `modules.GetModules().BuildType()` which can be Enterprise or OSS. Maybe we should just add a third option for community there?

Reply:
I like the idea. I'm not sure how it would work with updating with the build flags. Where are we setting this build type so that it can be overriden?  I took a stab at it here https://github.com/gravitational/teleport/pull/41294/commits/0294fef7998588eca304ba4c3d91b066ae27a29c#diff-8e9da4d6acbdd048ce7ceafde7ee7934d51890dc15475a71644a7a76f901884dR364

If the build flag isn't set, then it returns `oss`. And then `enterprise` pulls from the enterprise modules instead and will always be `ent` regardless if the build flag is set or not (it wont be set for `e` but just sayin).